external/fv3fit/fv3fit/sklearn/_random_forest.py

```diff
@@ -205,24 +212,22 @@ def __init__(
             output_variables: list of output variables
             model: a scikit learn regression model
         """
-        self._sample_dim_name = sample_dim_name
-        self._input_variables = input_variables
-        self._output_variables = output_variables
         self.model = model
-
         self.scaler_type = scaler_type
         self.scaler_kwargs = scaler_kwargs or {}
         self.target_scaler: Optional[scaler.NormalizeTransform] = None
+        # TODO: remove internal sample dim name once sample dim is hardcoded everywhere
+        self._sample_dim_name = sample_dim_name
```

Comment:
I think you can remove this `self._sample_dim_name` variable now, and remove it from the arguments in your later PR which will change the config.